__tests__/app.test.js

```diff
@@ -1,15 +1,66 @@
-const endpointsJson = require("../endpoints.json");
+const endpointsJson = require("../endpoints.json")
 /* Set up your test imports here */
-
+const request = require("supertest")
+const app = require("../app")
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const testData = require("../db/data/test-data")
 /* Set up your beforeEach & afterAll functions here */
 
+afterAll(() => {
+  return db.end()
+})
+
+beforeEach(() => {
+  return seed(testData)
+})
+
 describe("GET /api", () => {
   test("200: Responds with an object detailing the documentation for each endpoint", () => {
     return request(app)
       .get("/api")
       .expect(200)
       .then(({ body: { endpoints } }) => {
-        expect(endpoints).toEqual(endpointsJson);
-      });
-  });
-});
+        expect(endpoints).toEqual(endpointsJson)
+      })
+  })
+})
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of objects th have property os slug and description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics).toHaveLength(3)
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          })
+        })
+      })
+  })
+})
+test("200: the corrsponding response of object should be", () => {
+  return request(app)
+    .get("/api/topics")
+    .expect(200)
+    .then(({ body: { topics } }) => {
+      expect(topics).toHaveLength(3)
+      topics.forEach((topic) => {
+        expect(topics[0]).toMatchObject({
+          slug: "mitch",
+          description: "The man, the Mitch, the legend",
+        })
+      })
+    })
+})
+test("404: Responds with Sorry Not Found when requesting was from a bad URL", () => {
+  return request(app)
+    .get("/api/to")
```

Comment:
It may be worth having a more semantic name for your endpoint here, such as "not-an-endpoint" for example.